src/providers/Theme.vue

```diff
@@ -18,6 +18,13 @@ export default defineComponent({
 })
 </script>
 
+<style lang="scss">
+@import 'src/styles/reset.scss';
+[id^='humm-widgets-cards'] {
+  @extend %reset;
+}
```

Comment:
I actually think you can move the whole rule inside of `reset.scss` and not use the `%reset` psuedo class anymore – since the approach now is to avoid scoping for the resets. If I remember correctly we only needed to extend the styles like this previously when we wanted scoped resets, but that turned out to be too specific.

Reply:
nice! will include that fix.